src/components/SingleMovie/SingleMovie.js

```diff
@@ -1,15 +1,16 @@
 import React from 'react';
 import './SingleMovie.css';
+import {NavLink} from 'react-router-dom';
 
-const singleMovie = ({movie, method}) => {
+const SingleMovie = ({movie}) => {
```

Comment:
If you wanted to, you could use further destructuring to pull the keys out of the movie object - it could make your render a lil cleaner with less `movie.____` again and again